### ISSUE TITLE
File permissions doesn't explicitly set

### ISSUE THREAD

**Issue by ArseniiPetrovich** [poanetwork/deployment-bridge#8]
Lets look into one of the plays:
```
- name: "Bridge - create password files"
  template:
    src: "{{ item }}.j2"
    dest: "{{ bridge_path }}/{{ item }}"
  with_items:
    - foreign-password.txt
    - home-password.txt
  notify:
    - restart bridge
```
It creates a password file on a remote machine with a secret key which will be used by `bridge` service. If defaults doesn't set correcly it might lead to insecure, when hacker could easily get access to this file. My proposal is to explicitly grant file permissions. Also, if it supposed that this file will not be writable - we should set permissions like 400 to for the best secureness.

**Comment by phahulin** [comment]
Actually, I don't think that password file should be write-protected, since it's readable anyway

**Comment by ArseniiPetrovich** [comment]
Hmm. Look, here is an example scenario. I might be wrong, but it is kind of idea, that hacker could come up with when found out that file is not write-protected.
He can create his own crypto-wallet and replace all created passwords with his owns, so any coins that should go to your wallet will go to his.

**Comment by phahulin** [comment]
In our case, bridge validators can receive funds only on specific address, so this scenario is unlikely

**Comment by ArseniiPetrovich** [comment]
Resolved at #15 PR
